docs/user/supported_devices.rst

```diff
@@ -224,6 +224,15 @@ ipq806x-generic [#80211s]_
 
   - R7800
 
+
+lantiq-xway
+-----------
+
+* AVM
+
+  - Fritz!Box 7312 [#avmflash]_
```

Comment:
```suggestion
  - FRITZ!Box 7312 [#avmflash]_
```

Reply:
Actually - we made this mistake at some places already - just like the inconsistent whitespacing. I would merge this as it is and clean up both points after - Is this okay with you?

Reply:
Fine with me.